spec/fabricators/keyword_fabricator.rb

```diff
@@ -2,5 +2,6 @@
 
 Fabricator(:keyword) do
   keyword FFaker::FreedomIpsum.word
-  user Fabricate(:user)
+  status 0
```

Comment:
How about using `:in_progress` instead of `0` to make it more readable/understandable?
```suggestion
  status :in_progress
```

Reply:
Fixed in [`f0991a5` (#42)](https://github.com/suho/google-search-ruby/pull/42/commits/f0991a50f82121eb55cc6afc300dc953406ebac0)